bank_subscription_prediction/configs/baseline.yaml

```diff
@@ -0,0 +1,48 @@
+# Baseline experiment configuration
+
+# environment configuration
+settings:
+  docker:
+    required_integrations:
+      - sklearn
+      - pandas
+    requirements:
+      - matplotlib
+      - xgboost
+      - seaborn
+      - plotly
+      - jupyter
```

Comment:
could we just get this from the requirements.txt instead?

Reply:
im rather a fan of keeping this here, as then I can put zenml in the requirements.txt + this could be bespoke for each pipeline so the good practice is just to keep it here